tasks/utils.py

```diff
@@ -151,57 +151,47 @@ def get_git_branch_name():
     return check_output(["git", "rev-parse", "--abbrev-ref", "HEAD"]).strip()
 
 
-def query_version(ctx, git_sha_length=7, match=None):
+def query_version(ctx, git_sha_length=7, dca=False):
```

Comment:
What do you think about replacing the `dca` option by a  `prefix` one. This way if/when we add new agents/versions to the repo we don't have to change the logic again. I have the puppy agent or process/APM agent in mind.
This would give every project more flexibility. 

A `None` prefix would default to the main agent like right now.

Reply:
Yes great idea